docs/DeveloperGuide.adoc

```diff
@@ -775,8 +914,12 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  New functions must be added regularly.
+.  The system should work on both windows and unix.
+.  The system should respond within 3 seconds.
```

Comment:
3 seconds is pretty arbitrary. Does it mean 3.1 seconds is no good? 
Maybe try phrasing it something along the lines of `The system should be responsive to user inputs`

Reply:
noted, will do the necessary changes, thanks!